### DISCUSSION TITLE
Migration of tests from .xml format to .side

### DISCUSSION THREAD

**Discussion by alex270795** [SeleniumHQ/docker-selenium#2471] [Q&A]
Dear,
Now we want to migrate from Selenium IDE 2.x to Selenium IDE 4.x .
Is it a way to migrate tests from an old IDE to new one?
I want to mention that old tests we have in .xml format and these tests are not working properly on new IDE 4.x .
I've seen that there were an npm package selenium_ide_import , but now it's not accessible.
So, is there any analogs of selenium_ide_import ?

BR,
Alexandru.

**Comment by diemol** [comment]
I am unsure this is possible, but perhaps you can join the IDE channel in our Slack https://www.selenium.dev/support/#ChatRoom

This repo is only for Docker Selenium discussions.
